config/_default/menus/menus.fr.yaml

```diff
@@ -724,24 +719,24 @@ main:
     parent: tracing_vis
     weight: 405
   - name: Analyse et recherche de traces
-    url: tracing/app_analytics/
+    url: tracing/trace_search_and_analytics/
```

Comment:
@l0k0ms - do I need to go into transifex to update these? right now, merging this in will break the links again

Reply:
I'm on it :)